src/HealthCheck.Mvc/Controllers/MvcHealthCheckController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Remoting.Messaging;
+using System.Web.Mvc;
+using HealthCheck.Core;
+using HealthCheck.Mvc.Models;
+
+namespace HealthCheck.Mvc.Controllers
+{
+    public class MvcHealthCheckController : AsyncController
```

Comment:
Don't inherit from `AsyncController`, instead implement async controller method as described in [the ASP.NET MVC 4 docs](https://docs.microsoft.com/en-us/aspnet/mvc/overview/performance/using-asynchronous-methods-in-aspnet-mvc-4) (using `Task<ActionResult>`).